va/va.go

```diff
@@ -39,6 +39,9 @@ const (
 	// allowed accept up to 128 bytes before rejecting a response
 	// (32 byte b64 encoded token + . + 32 byte b64 encoded key fingerprint)
 	maxResponseSize = 128
+
+	tlsSNITokenID = "token"
+	tlsSNIKaID    = "ka"
```

Comment:
both `tlsSNITokenID` and `tlsSNIKaID` are only used in `validateTLSSNI02`. Do you think its better to inline them there? I'm not sure there will be a case where we'd want to reference these outside of the validate method.